src/libraries/System.Runtime/tests/TrimmingTests/InvariantGlobalizationFalse.cs

```diff
@@ -0,0 +1,26 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
>// [](http://example.com/codeflow?start=0&length=2)

do you think we need to have any test when enabling NLS mode on Windows?

Reply:
Do we have a feature switch for UseNls? Does that property participate in trimming at all?

Reply:
> Do we have a feature switch for UseNls? 

Yes, https://github.com/dotnet/runtime/blob/f881cf3839d44fc356dc3a4dd1c1b7b19cc82b66/src/libraries/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.Windows.cs#L12.

> Does that property participate in trimming at all?

I don't think so. I am trying to ensure Invariant and NLS behaviors modes not affected. I was just asking if we need to do something more here. We already have tests specific for NLS which should be covering that. If you think nothing more need to be done, you can ignore this comment.

Reply:
I think the current tests will suffice.